app/views/status_messages/new.html.haml

```diff
@@ -7,7 +7,8 @@
 :javascript
   $(function() {
     app.publisher = new app.views.Publisher({
-      standalone: true
+      standalone: true,
+      prefillMention: #{@person.to_json}
```

Comment:
Hmm... `#{html_escape(@person.to_json)}` also breaks the prefill mention. Not sure how I can protect here.